lambda/src/main/resources/db/migration/V126__update_ffidmetadatamatches_to_include_mismatch_boolean_and_format_name.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "FFIDMetadataMatches"
```

Comment:
These new columns can be optional, so should be allows to be `NULL`

Also shouldn't provide the default value of `FALSE` as for existing entries we don't actually know if the it wasn't a file extension mismatch.